.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build
+on:
+   push:
+     branches: [ 'main' ]
+     paths-ignore: [ 'docs/**' ]
+
+permissions: write-all
+
+env:
+  VERSION: 0.0.1
+
+jobs:
+  avalonia:
+    name: Fabulous.Avalonia
+    runs-on: macos-12
+    env:
+      SLN_FILE: Fabulous.Avalonia.NoSamples.sln
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 7.0.x
+    - name: Restore
```

Comment:
I'm guessing you would need to install the dotnet workloads iOS and Android since you are targeting `net7.0-android` and `net7.0-ios`

Reply:
Should be something like `dotnet workload install android ios`

Reply:
Same applies for all CI files